src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingExerciseRepositoryService.java

```diff
@@ -312,6 +316,11 @@ private void setupJVMTestTemplateAndPush(final RepositoryResources resources, fi
             setupJVMTestTemplateProjectTypeResources(resources, programmingExercise, repoLocalPath);
         }
 
+        if (projectType == ProjectType.MAVEN_BLACKBOX) {
+            Path dejagnuLibFolderPath = repoLocalPath.resolve("dejagnu").resolve("lib");
```

Comment:
```suggestion
            Path dejagnuLibFolderPath = repoLocalPath.resolve("testsuite").resolve("lib");
```
That changed with Dejagnu 1.6.3, I think?

Reply:
True, should be part of the fixed I just pushed. Thanks for noticing.